ui_src/src/domain/administration/integrations/components/dataDogIntegration/style.scss

```diff
@@ -1,66 +1,67 @@
-.integration-guid-stepper{
+.integration-guid-stepper {
     width: 95%;
     align-self: center;
     margin-top: 10px;
     height: calc(100% - 420px);
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    .ant-collapse{
+    .ant-collapse {
         background: white;
-        border: 1px solid #E6E6E6;
+        border: 1px solid #e6e6e6;
         box-shadow: 0px 0px 2px rgb(24 24 28 / 2%), 0px 1px 2px rgb(24 24 28 / 6%);
         border-radius: 8px;
         max-height: calc(100% - 70px);
         overflow: auto;
-        .ant-collapse-header{
-            flex-flow:row-reverse;
+        .ant-collapse-header {
+            flex-flow: row-reverse;
             padding: 9px 16px;
-            .ant-collapse-header-text{
+            .ant-collapse-header-text {
                 font-family: 'InterMedium';
                 font-size: 15px;
             }
         }
-        .ant-collapse-content-box{
+        .ant-collapse-content-box {
             padding: 0;
         }
-        .steps-content{
+        .steps-content {
             max-height: 240px;
             overflow: auto;
             padding: 16px;
-            h3{
+            h3 {
                 color: #727279;
-                font-size:14px;
+                font-size: 14px;
             }
-            p{
+            p {
                 font-family: 'InterMedium';
                 font-size: 14px;
                 color: #222124;
                 margin: 0;
             }
-            span{
+            span {
                 color: #727279;
-                font-size:12px;
+                font-size: 12px;
             }
-            label{
+            label {
                 color: rgba(101, 87, 255, 0.8);
                 background: rgba(101, 87, 255, 0.2);
                 border-radius: 4px;
                 padding: 3px;
             }
-            a{
+            a {
                 color: var(--purple);
             }
-            a:hover{
+            a:hover {
                 text-decoration: underline;
             }
-            .img{
+            .img,
```

Comment:
the same

Reply:
It is needed in the index file. it is used in png file